content/faq/how-to-publish.md

```diff
@@ -119,7 +119,7 @@ Yes, the claim can be edited to increase the bid amount or you can also send a S
 
 ## My video doesn't stream in the app or on lbry.tv, what's wrong?
 
-The in-app and lbry.tv video player's streaming capability works best with the H264/AAC format, typically in a MP4 container but others like M4V will also work. Other non-streaming video types like AVI/WMV/MOV file formats are not supported by the in-app player, but they can be shared/downloaded and will play outside of the app.
+The in-app and lbry.tv video player's streaming capability works best with the H264/AAC format, typically in an MP4 container but others like M4V will also work. Other non-streaming video types like AVI/WMV/MOV file formats are not supported by the in-app player, but they can be shared/downloaded and will play outside of the app.
```

Comment:
The only time you use "an" is if the after part is a vowel, A, E, I, O, U

Reply:
This is not true. On acronyms you have to see what the first letter is. If it's M you pronounce it "emm" and put an in front of it.

Reply:
Didn't know this, thanks, sorry.